django/contrib/auth/forms.py

```diff
@@ -190,11 +190,14 @@ def clean(self):
         password = self.cleaned_data.get('password')
 
         if username is not None and password:
+            try:
```

Comment:
Would it be better to do the query after `authenticate()`? Otherwise, this adds a query for all login attempts instead of only failed attempts.

For the commit message, use:
```
Fixed #28645 -- Reallowed AuthenticationForm to raise the inactive user error when using ModelBackend.

Regression in e0a3d937309a82b8beea8f41b17d8b6298da2a86.
```